src/main/java/com/denizenscript/depenizen/bukkit/events/crackshot/crackshotLandmineTriggerEvent.java

```diff
@@ -0,0 +1,80 @@
+package com.denizenscript.depenizen.bukkit.events.crackshot;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.shampaggon.crackshot.events.WeaponTriggerEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class crackshotLandmineTriggerEvent extends BukkitScriptEvent implements Listener {
```

Comment:
All Java classes must ALWAYS start with a capital letter

Reply:
This should be corrected now.